src/cssm.pyx

```diff
@@ -870,7 +870,155 @@ def ddm_flex(np.ndarray[float, ndim = 1] v,
     else:
         raise ValueError('return_option must be either "full" or "minimal"')
 # ----------------------------------------------------------------------------------------------------
+def ddm_flex_no_v(
```

Comment:
question here: was there no way to reuse `ddm_flex` by e.g. setting `v` to `0`?

Reply:
I think the issue was that there's no way to set "v" equal to 0 in hssm. I also would need to specify bounds in config for "v" even though I don't want it to be simulated. I could try to add an if statement in simulator.py that just sets theta["v"] = 0, though id still have to specify the bounds and "estimate" it in hssm.